tests/post/steps/test_liveness.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from pytest_bdd import scenario, given, when, then, parsers
+
+# Scenarios
+@scenario('../features/pods_alive.feature', 'get pods')
+def test_pods(host):
+    pass
+
+# Given
+
+@given(parsers.parse("The '{service_name}' service is up"))
+def check_service(host, service_name):
+   assert host.service(service_name).is_running
+
+
+# Then
+@then(parsers.parse("the '{resource}' list should not be empty in the '{namespace}' namespace"))
+def check_resource_list(host, resource, namespace):
+    cmd = ("sudo kubectl --kubeconfig=/etc/kubernetes/admin.conf"
+           " get {0} --namespace {1} -o custom-columns=:metadata.name")
+    cmd_res = host.run(command=cmd.format(resource, namespace))
```

Comment:
Use https://testinfra.readthedocs.io/en/latest/modules.html#testinfra.host.Host.check_output instead, which will raise if the command failed (which we want, I guess?), and simply return the command's output.